features/step_definitions/api-token.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const jwt = require('jsonwebtoken');
+const request = require('../support/request');
+
+/**
+ * Clean up: remove the test token
+ * @method
+ * @param  {String} tokenName
+ * @return {Promise}
+ */
+function cleanup(tokenName) {
+    return request({
+        uri: `${this.instance}/${this.namespace}/tokens`,
+        method: 'GET',
+        auth: {
+            bearer: this.jwt
+        }
+    }).then((response) => {
+        const match = JSON.parse(response.body)
+            .find(token => token.name === tokenName);
+
+        if (!match) return Promise.resolve();
+
+        return request({
+            uri: `${this.instance}/${this.namespace}/tokens/${match.id}`,
+            method: 'DELETE',
+            auth: {
+                bearer: this.jwt
+            }
+        }).then(deleteResponse => Assert.strictEqual(deleteResponse.statusCode, 204));
+    });
+}
+
+module.exports = function server() {
+    this.After(() => {
+        if (this.testToken) {
+            return cleanup.apply(this, [this.testToken.name]);
```

Comment:
Ahh, good to know about! Actually switched to using a `config` object like the other functions in `support` though, since the function got moved to `support` to be accessible to the `after` hook.